src/libsyntax/parse/parser.rs

```diff
@@ -5066,18 +5066,25 @@ impl<'a> Parser<'a> {
     fn parse_struct_decl_field(&mut self) -> PResult<'a, StructField> {
         let attrs = self.parse_outer_attributes()?;
         let lo = self.span.lo;
-        let vis = self.parse_visibility()?;
+        let vis = self.parse_visibility(true)?;
```

Comment:
Ah, that reminds me. Seems like we should have some tests exercising these paths (i.e., tests that would fail with this being `true`)...